api/Element.json

```
@@ -6207,7 +6207,8 @@
               "version_added": "22"
             },
             "ie": {
-              "version_added": "11"
+              "version_added": "8",
+              "notes": "Internet Explorer 10 and below won't trigger paste event when copying files."
```

Some stylistic suggestions (e.g., using the present tense)

```suggestion
              "notes": "Before Internet Explorer 11, copying files does not trigger the <code>paste</code> event."
```